include/core/truthTable/truth_table.hpp

```diff
@@ -330,6 +329,25 @@ namespace syrec {
             cubeMap.insert(std::move(nh));
         }
 
+        static auto equal(TruthTable& ttOriginal, TruthTable& ttSimOut, const bool equalityUpToDontCare = true) -> bool {
```

Comment:
```suggestion
        static auto equal(TruthTable& tt1, TruthTable& tt2, const bool equalityUpToDontCare = true) -> bool {
```
Please try to name variables appropriately. The parameters here have nothing to do with "original" or "simulation" at all.